package.json

```diff
@@ -10,6 +10,7 @@
   },
   "license": "MIT",
   "scripts": {
+    "postinstall": "rm -rf static/bin && cp -rp node_modules/@mysteriumnetwork/node/bin static/bin",
```

Comment:
This will not work on windows. I could think of creating js script and run it here as `node postinstall.js` or investigate other solutions.

Reply:
This PR scopes just MacOS building, Solving Windows build flow in upcoming PR